spec/acceptance/netapp_license_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper_acceptance'
+
+describe 'netapp_license' do
+  it 'add a netapp_license' do
+    pp=<<-EOS
+node 'vsim-01' {
+  netapp_license { 'snaprestore' :
+    ensure => present,
+    codes  => "saiufbaisubfiuasbfiuabs",
+  }
+}
+node 'vserver-01' {
+}
+    EOS
+    make_site_pp(pp)
+    run_device(:allow_changes => true)
+    run_device(:allow_changes => false)
```

Comment:
I assume this wouldn't actually work? Does the API raise failures, or just continue to change?